### DISCUSSION TITLE
Rendszerhatáron belüli fogalom értelmezése

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#47] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők!

Még a korábbi dokumentációban jött ellő a rendszerhatáron belüli fogalom, mely a bevallás helyesbítésével (javítás/önellenőrzés) kapcsolatban merült fel. A jelenlegi elképzelések szerint ezt majd csak rendszerhatáron belül lehet megtenni.

Ezzel kapcsolatban lenne két kérdésünk/ észrevételünk:
1. Ez nem fogja kitolni nagyon az ÁNYK kivezetésének az időpontját? 
2. Ha kivezetésre kerül az ÁNYK, akkor az ÁNYK-val benyújtott bevallások önellenőrzése, hogy fog történni?

Érdemes megfontolni az alábbi szcenáriót is:
- Tegyük fel, hogy az adózó ERP rendszert vált.
- Korábban a bevallását ÁNYK-ban adta be, de az ÁNYK-t kivonták a forgalomból, leváltotta az eÁFA rendszer.
-  Az új ERP rendszer megnyitása a nyitó adatokkal történik, a történeti adatok átvételére nincs lehetőség a tárolókapacitások korlátozottsága miatt.
- Kérdés: Hogyan fogja tudni megtenni az adózó az önellenőrzés az ÁNYK-ban beadott időszakokra?

Előre is köszönjük a válaszokat!

**Comment by renced42** [comment, marked as answer]
Kedves @szalaierika 

Az eÁFA rendszerben jelenleg nem lehet önellenőrzést beadni, de ez a funkció is el fog indulni hamarosan.
eÁFA rendszerben csak eÁfa rendszerben beadott bevallást lehet első körben önellenőrizni. (Ez nyilvánvalóan változni fog ha kötelező lesz az eÁFA használata.)

ÁNYK-n keresztül pedig be lehet adni az önellenőrzést mind eÁFA-n mind ÁNYK-n keresztül beadott bevallásokra átmeneti időszakra.
Amennyiben az ÁNYK tekintetében a 65-ös bevallás beadására nem lesz lehetőség, akkor az ügyfélnek ugyanolyan szcenárió mentén kell az önellenörzést elkészítenie, mintha nem lenne eÁFA rendszer. Ez a problématér ettől független. 
Viszont az ügyfélnek lehetősége lesz a NAV-nál rendelkezésre álló adatokat lekérdezni. Számlák bizonylatok, bevallások.
